controllers/gce/healthchecks/healthchecks.go

```diff
@@ -19,76 +19,198 @@ package healthchecks
 import (
 	"net/http"
 
-	"github.com/golang/glog"
 	compute "google.golang.org/api/compute/v1"
 
+	"github.com/golang/glog"
+
 	"k8s.io/ingress/controllers/gce/utils"
 )
 
+const (
+	// These values set a low health threshold and a high failure threshold.
+	// We're just trying to detect if the node networking is
+	// borked, service level outages will get detected sooner
+	// by kube-proxy.
+	// DefaultHealthCheckInterval defines how frequently a probe runs
+	DefaultHealthCheckInterval = 60
```

Comment:
I agree that this would be desirable. However, the compute api takes int64 of seconds. If we made this a time.Duration, we would need to `int64(DefaultHealthCheckInterval.Seconds())` which isn't ideal.

Should we settle for renaming to `DefaultHealthCheckIntervalSeconds`?